src/prpy/planning/base.py

```diff
@@ -298,6 +318,23 @@ def plan(self, method, args, kw_args):
                     with Timer() as timer:
                         output = planner_method(*args, **kw_args)
 
+                    if not is_sequence_deterministic:
+                        # TODO: It is overly conservative to set _ENDPOINT,
+                        # e.g. for PlanToConfiguration. Unfortunately, there is
+                        # no easy way to detect this special case.
+                        SetTrajectoryTags(output, {
+                            Tags.DETERMINISTIC_TRAJECTORY: False,
+                            Tags.DETERMINISTIC_ENDPOINT: False,
```

Comment:
This condition is triggered by _previous_ plans in the `Sequence` failing. We would have to check whether all previous planners that failed _would have_ had a deterministic endpoint _had they succeeded_.

We could theoretically implement that by adding an `endpoint_deterministic` flag to `PlanningError`. However, it's a bit too counterfactual for my taste.